app/modules/net.c

```diff
@@ -983,7 +983,10 @@ field_from_ipaddr(lua_State *L, const char * field_name, ip_addr_t* addr) {
 
 static int net_ifinfo( lua_State* L ) {
   int ifidx = luaL_optint(L, 1, 0);
-
+  if (ifidx!=0x00 && ifidx!=0x01) {
```

Comment:
```suggestion
  if (ifidx != 0 && ifidx != 1) {
```

This assertion is in line with what the documentation has been stating all along 😉